### DISCUSSION TITLE
Edit worskspace.yaml of already running (dockerized) dagit instance.

### DISCUSSION THREAD

**Discussion by KWiecko** [dagster-io/dagster#3545] [Q&A]
Dear all,

we are trying to deploy our data apps with dagit via docker-compose (one service for dagit, one service for dagit's postgres to store e.g. schedule metadata and one service with virtualenv per each application / repository we want to run). 
Now with folder mounting / binding done correctly in docker-compose we can easily add new pipelines do already existing repository (which has been listed in workspace.yaml prior to dagit's start). 
However if we want to add new repository to the workspace.yaml dagit does not seem to notice those changes. (workspace.yaml is in mounted /bound docker's folder so we can change it from 'outside' of the container any time we want)

TL;DR:
Is there a way to make dagit notice changes made to its workspace.yaml after it has been started ? We changed workspace.yaml with dagit running and it didn't seem to notice any changes.

All the best,
Konrad.
